### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by daniil**
- Add footer with copyright sign

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,5 +10,8 @@
     <h1>Amazing App &trade;</h1>
     <h2>Most amazing app ever</h2>
   </header>
+  <footer>
+    &copy; 2023 - Best App Ever
```
Comment (bstn-june-2023-toronto) [inline index.html:14]: Can we wrap this in a span and make year bolded

**Comment by bstn-june-2023-toronto** [comment]
We should add social links as part of requirements

File: index.html (hunk 1 above)
Comment (daniil) [reply index.html]: I think span is unnecessary, but bolded as requested

Comment (bstn-june-2023-toronto) [reply index.html]: That makes sense. Looks good.

**Comment by bstn-june-2023-toronto** [comment]
Everything looks good. 🚢 it

**Pull request merged**